src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -74,6 +74,7 @@ export const BuilderApplet = () => {
   useEffect(
     () => () => {
       builderSessionStorage.removeItem();
+      dispatch(applet.actions.removeApplet());
```

Comment:
don't you need an argument inside of removeApplet()?

Reply:
no need, we get rid of the state of applet (we have only current single applet data) got on the builder page